src/charm.py

```diff
@@ -27,7 +27,6 @@
 
 UPF_SNAP_NAME = "sdcore-upf"
 UPF_SNAP_CHANNEL = "1.4/edge"
-UPF_SNAP_REVISION = "49"
```

Comment:
On the other hand, whenever new snap is released we need to remember about about updating the charm. Shouldn't we be controlling snap version through the channels instead of revision?

Reply:
The problem is that if we don't pin the revision somehow in here, we might have no way of reproducing the deployment. Let's say someone deploys from 1.4/stable, and we push an update that we think is stable, that user would have no way of redeploying the same UPF that they are currently using. The charm would not update the snap in place, but getting an exact replica of the environment would be possible without manual intervention.

Reply:
I agree with @ghislainbourgeois here, and even if we want to make this change, let's separate it from the dpdk PR.

Reply:
Reverted and bumped to the latest version.